### PR TITLE
add bd-xray plugin

### PR DESCRIPTION
**Pull request by yashbhutwala**
Allows developers/operators to scan already deployed images as well as about to be deployed images for open source security and license compliance using Black Duck by Synopsys.

### CODE REVIEW

File: plugins/bd-xray.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          arch: amd64
+      uri: "https://github.com/blackducksoftware/kubectl-bd-xray/releases/download/v0.1.1/kubectl-bd-xray_v0.1.1_windows_amd64.zip"
+      sha256: f454b9c5e84ee1b004b072cbca0b3ff982d1578a45efab4cbe798bc97d06dece
+      files:
+        - from: "bd-xray.exe"
+          to: "."
+        - from: "LICENSE"
+          to: "."
+      bin: "bd-xray.exe"
+  shortDescription: Run Black Duck Image Scans for software composition analysis.
```
Comment (ahmetb) [inline plugins/bd-xray.yaml:44]: If you can go shorter, it'll fit to the screen without being truncated e.g.

> Run BlackDuck image scans on pods.

File: plugins/bd-xray.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  homepage: https://github.com/blackducksoftware/kubectl-bd-xray
+  description: |
+    This plugin runs Black Duck Image Scans for software composition analysis.
+    Just point and scan images in any namespace, third-party or your own yaml files, helm charts and more.
```
Comment (ahmetb) [inline plugins/bd-xray.yaml:48]: If you can line wrap at 80 chars, it would better fit the terminals.

File: plugins/bd-xray.yaml (hunk 1 above)
Comment (yashbhutwala) [reply plugins/bd-xray.yaml]: made this shorter.

File: plugins/bd-xray.yaml (hunk 2 above)
Comment (yashbhutwala) [reply plugins/bd-xray.yaml]: done, thanks!

**Pull request merged**